data/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.data">
+
+</manifest>
```

Comment:
add an empty line at the end of the file pls

Reply:
done!